emojify.py

```diff
@@ -5,16 +5,20 @@
 EMOJI_BLOCK_SZ = 32
 
 def main(argc, argv):
-    if argc < 4:
-        print '%s <input gif> <output directory> <emoji name> <max width emojis>' % argv[0]
+    if argc < 5:
+        print '%s <input gif> <output directory> <emoji name> <max width emojis> [max height emojis]' % argv[0]
         return 1
     
     try:
         if not os.path.exists(argv[2]):
             os.makedirs(argv[2])
         max_w = int(argv[4])
         clip = VideoFileClip(argv[1])
-        max_h = int(math.floor(MAX_EMOJI_COUNT / max_w))
+        max_h = 0
+        if argc == 6:
+            max_h = int(argv[5])
```

Comment:
The user should be warned if `max_w*max_h` goes past `MAX_EMOJI_COUNT`. The limit is enforced since that's the max emoji limit on servers

Reply:
Will do, thanks